lib/sles4sap/sdaf_deployment_library.pm

```diff
@@ -827,6 +828,48 @@ sub get_sdaf_deployment_command {
     return $cmd;
 }
 
+=head2 git_clone
+
+    git_clone('https://github.com/myrepo/tree/main' [, branch=>'development', quiet=>'1', skip_ssl_verification=>'true',
+        output_log_file=>'git_clone.log']);
+
+B<repository>: Git repository url. Mandatory argument.
+
+B<branch>: Clone specific branch. Default: not defined
+
+B<quiet>: Minimize output verbosity. Default: false
+
+B<skip_ssl_verification>: Enable or disable SSL related error. Useful in case of self signed certificates. Default: false
+
+B<output_log_file>: Log output into a file.
+
+Generic wrapper around C<git clone> command. Supports basic set of switches and output logging.
+
+=cut
+
+sub git_clone {
+    my ($repository, %args) = @_;
+    croak 'Missing mandatory argument "repository"' unless $repository;
+
+    # Base command
+    my $git_cmd = 'git clone';
+
+    # Skip SSL verification
+    $git_cmd =~ s/git clone/git -c http.sslVerify=false clone/ if $args{skip_ssl_verification};
```

Comment:
What about 
``` $git_cmd =~ s/git/git -c http.sslVerify=false/ if $args{skip_ssl_verification};```

Since I don't want to overwrite existing command (which even my original code did), but only add one option.